script/repl

```diff
@@ -11,5 +11,5 @@ repl_builddir=dist-repl
 if [[ ! -d $repl_builddir ]]; then
   echo "$repl_builddir does not exist, first run 'cabal repl --builddir=$repl_builddir', exit, and then re-run $0"
 else
-  cabal exec --builddir=$repl_builddir ghci -- -ghci-script=.ghci.semantic $@
+  cabal exec --builddir=$repl_builddir ghci -- -ghci-script=.ghci.semantic $(script/ghci-flags --builddir "$repl_builddir") -no-ignore-dot-ghci $@
```

Comment:
1. We reuse `script/ghci-flags` from `script/repl` in order to share a bunch of the path computation logic.

2. `script/ghci-flags` tells `ghcide` to ignore `.ghci` files so as to avoid getting confused, so we also say `-no-ignore-dot-ghci` afterwards because `script/repl` _should_ load them.